### DISCUSSION TITLE
How to use nr-binder in ueransim container

### DISCUSSION THREAD

**Discussion by YzkkkkkNB** [aligungr/UERANSIM#627] [Q&A]
First，I was able to successfully transmit our python files using the nr-binder by the official UERANSIM program
![image](https://user-images.githubusercontent.com/129350910/228721897-d0e00bfb-63e7-4237-8c79-5e114a7cdea5.png)

Then，I use the UERANSIM image to develop a ueransim container. However, in the container the nr-binder is unenforceable
![image](https://user-images.githubusercontent.com/129350910/228722386-d351a359-54a8-4fca-b905-77990937b66a.png)
After changing the file executable permissions, python files still cannot be transferred using nr-binder.
Why? I wonder if it's because the nr-binder in the container is not functioning fully

**Comment by Noufel17** [comment]
I got the same issue
